managed/models/database.go

```diff
@@ -728,6 +728,22 @@ var databaseSchema = [][]string{
 		)`,
 	},
 	66: {
+		`UPDATE agents
+		SET log_level = 'error'
+		WHERE log_level = 'fatal'
+		AND agent_type IN (
+			'node_exporter',
+
+			'mysqld_exporter',
+			'qan-mysql-perfschema-agent',
+			'qan-mysql-slowlog-agent',
+
+			'postgres_exporter',
+			'qan-postgresql-pgstatements-agent',
+			'qan-postgresql-pgstatmonitor-agent'
+		);`,
+	},
+	67: {
```

Comment:
Bad merge with `main` branch. Your migration should have number `67`

Reply:
done